docs/tools/mbed_targets.md

```diff
@@ -148,6 +149,45 @@ The build system errors when you use features outside of this list.
 
 When you use target inheritance, you may alter the values of `features` using `features_add` and `features_remove`. This is similar to the `macros_add` and `macros_remove` mechanism the previous section describes.
 
+#### `config` and `overrides`
+
+The list of _configs_ provide a way to modify the values of macros in child targets or in a project. Each configuration has a macro name, as well as a default value and an optional help value. For example:
+
+```json
+"config": {
+    "clock_src": {
+        "help": "Clock source to use, can be XTAL or RC",
+        "value": "XTAL",
+        "macro_name": "CLOCK_SRC"
+    }
+}
+```
+
+ This case defines the config `clock_src`, for the `CLOCK_SRC` macro, and has a default value of `XTAL`.
```

Comment:
wouldn't be easier to show here default naming convention for config and also mention that macro_name overrides it ?

Reply:
ah, I didnt even realize it defaults to the config name.